plugins/EpadDB.js

```diff
@@ -2209,36 +2339,129 @@ async function epaddb(fastify, options, done) {
 
             const dock = new DockerService(fs, fastify, path);
             const containerName = `epadplugin_${pluginObj.id}`;
-            dock
+            // eslint-disable-next-line no-await-in-loop
+            await dock
```

Comment:
you cannot use both await and then. you should convert it to await structure (remove then and put an inner try catch for error handling)

Reply:
switched from then to await